### PR TITLE
now checks version string consistency

### PR DESCRIPTION
**Pull request by jspaaks**

### CODE REVIEW

File: tests/test_version_string_consistency.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import os
+import re
+from matchms import __version__ as expected_version
+
+
+def test_version_string_consistency():
+
```
Comment (fdiblen) [inline tests/test_version_string_consistency.py:8]: add docstrings

File: tests/test_version_string_consistency.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    module_root = os.path.join(os.path.dirname(__file__), '..')
```
Comment (fdiblen) [inline tests/test_version_string_consistency.py]: change module_root to repository_root

File: tests/test_version_string_consistency.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    fixture = os.path.join(module_root, "meta.yaml")
+
+    with open(fixture, "r") as f:
+        metayaml_contents = f.read()
+
+    match = re.search(r"^{% set version = \"(?P<semver>.*)\" %}$", metayaml_contents, re.MULTILINE)
+    actual_version = match["semver"]
+
+    assert expected_version == actual_version
```
Comment (fdiblen) [inline tests/test_version_string_consistency.py]: add a message for assert statement

**Review by fdiblen — CHANGES_REQUESTED** [review]
Suggested few changes please check my comments

File: tests/test_version_string_consistency.py (hunk 1 above)
Comment (jspaaks) [reply tests/test_version_string_consistency.py:8]: done

File: tests/test_version_string_consistency.py (hunk 2 above)
Comment (jspaaks) [reply tests/test_version_string_consistency.py]: done

File: tests/test_version_string_consistency.py (hunk 3 above)
Comment (jspaaks) [reply tests/test_version_string_consistency.py]: done

**Review by fdiblen — APPROVED** [review]
Great!

**Pull request merged**